### DISCUSSION TITLE
gateway는 무엇이고 어떤 역할을 할까?

### DISCUSSION THREAD

**Discussion by yeonkkk** [woowacourse-study/2023-tal-study#23] [Q&A] [answered]
강의를 보면 택배기사를 게이트웨이에 빗대어 표현하는데, 정확히 어떤 점 때문에 이런 비유를 하는 것인지 궁금합니다!

참고 PR: https://github.com/woowacourse-study/tal-study/pull/20/files

**Comment by kyY00n** [comment, marked as answer]
여기 게이트웨이 설명이 기가 막힙니다
https://rangerang.tistory.com/49

**Comment by zillionme** [comment]
이것에 대해 한번 더 논의 하고 싶어요!
